ci/windows/test_py2_cpu.ps1

```diff
@@ -27,3 +27,6 @@ C:\Python27\python.exe -m nose -v --with-timer --timer-ok 1 --timer-warning 15 -
 if (! $?) { Throw ("Error running unittest") }
 C:\Python27\python.exe -m nose -v --with-timer --timer-ok 1 --timer-warning 15 --timer-filter warning,error --with-xunit --xunit-file nosetests_train.xml tests\python\train
 if (! $?) { Throw ("Error running train tests") }
+$env:MXNET_SAFE_ACCUMULATION=1
```

Comment:
Add a comment here on what was tried and why we test it this way

Reply:
Done.